pulsar-broker-common/src/main/java/org/apache/pulsar/broker/BaseGenerateDocumentation.java

```diff
@@ -124,14 +124,10 @@ public int compare(Field o1, Field o2) {
     }
 
     protected String prefix = """
-            :::note
-
-            This page is automatically generated from code files.
+            !> This page is automatically generated from code files.
```

Comment:
what's this `!>`?

Reply:
Important content in Docsify. You can find it in Docify Docs.